src/components/provider.ts

```diff
@@ -60,16 +59,22 @@ export interface EIP1193Provider extends EventEmitter {
 
 const hooks: components.IHooks = {
   afterCopyStatic: [
-    async (dest, frm, component): Promise<void> => {
-      if (component.language === "typescript") {
-        return await move(path.join(dest, "_package.json"), path.join(dest, "package.json"), { overwrite: true });
+    // eslint-disable-next-line consistent-return
+    async (dest, _, component): Promise<void> => {
+      if (component.language === 'typescript') {
+        return await move(
```

Comment:
Nit: The "consistent return" rule wants all branches to return, or none. So this could be addressed by adding a `return undefined` at the end, but it's probably easier to drop the `return` instead since it'd be shorter and functionally equivalent

```suggestion
    async (dest, _, component): Promise<void> => {
      if (component.language === 'typescript') {
        await move(
```

Reply:
Similar case below on lines 74/86